packages/gapi-ocaml/gapi-ocaml.0.3.4/opam

```diff
@@ -28,9 +28,7 @@ depends: [
   "ocurl"
   "xmlm"
   "yojson"
-]
-depopts: [
-  "ounit" {build & test}
-  "pa_monad_custom" {build}
+  "ounit" {test}
+  "pa_monad_custom" {test}
```

Comment:
is it really only needed for the tests then?

Reply:
Yes, more or less. There are other targets that use that dependency but you cannot build them with opam anyway.